Error while checking build/O2/fullCI_slc9 for 39dab7eb655f97fe95ee94eb857d97e4b3f28c9c at 2025-05-03 13:53:
```
## sw/BUILD/O2Physics-latest/log
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:
Error in cling::AutoLoadingVisitor::InsertIntoAutoLoadingState:


## sw/BUILD/O2-sim-challenge-test-latest/log
./sim-challenge.logDetected critical problem in logfile tpcreco.log
./sim-challenge.logtpcreco.log:[9352:internal-dpl-ccdb-backend]: [13:53:34][ERROR] Exception while running: Fatal error. Rethrowing.
./sim-challenge.logtpcreco.log-[9352:internal-dpl-ccdb-backend]: [13:53:34][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-tpc-reco-workflow, device shutting down. Reason: Fatal error
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:33][ERROR] CCDBDownloader CURL transfer error - Timeout was reached
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:33][ERROR] CcdbDownloader finished transfer http://alice-ccdb.cern.ch/CTP/Calib/OrbitReset for 1546300800000 (agent_id: alimetal04.cern.ch-1746273211-zBgMym) with http code: 0
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:33][ERROR] File CTP/Calib/OrbitReset could not be retrieved. No more hosts to try.
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:33][FATAL] Unable to find CCDB object CTP/Calib/OrbitReset/1546300800000
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:34][ERROR] Exception while running: Fatal error. Rethrowing.
./sim-challenge.log[9352:internal-dpl-ccdb-backend]: [13:53:34][FATAL] Unhandled o2::framework::runtime_error reached the top of main of o2-tpc-reco-workflow, device shutting down. Reason: Fatal error
./sim-challenge.log[ERROR] Workflow crashed - PID 9352 (internal-dpl-ccdb-backend) did not exit correctly however it's not clear why. Exit code forced to 128.
./sim-challenge.log[ERROR]  - Device internal-dpl-ccdb-backend: pid 9352 (exit 128)
./sim-challenge.log[INFO]    - First error: [13:53:33][FATAL] Unable to find CCDB object CTP/Calib/OrbitReset/1546300800000
./sim-challenge.log[ERROR] SEVERE: Device internal-dpl-ccdb-backend (9352) had at least one message above severity 7: Unable to find CCDB object CTP/Calib/OrbitReset/1546300800000
./grp.log[ERROR] CCDBDownloader CURL transfer error - Timeout was reached
./grp.log[ERROR] CcdbDownloader finished transfer http://alice-ccdb.cern.ch//GLO/Calib/MeanVertex for 1546300800000 (agent_id: alimetal04.cern.ch-1746273041-XbF6dJ) with http code: 0
./grp.log[ERROR] File /GLO/Calib/MeanVertex could not be retrieved. No more hosts to try.
./grp.log[ERROR] Unable to find object /GLO/Calib/MeanVertex/1546300800000, Aborting
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/37}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/38}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/40}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/42}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/43}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/45}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/46}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/47}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/48}
./digi.log[ERROR] Found duplicate input binding with different spec.:collisioncontext {SIM/COLLISIONCONTEXT/49}
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/14233/39dab7eb655f97fe95ee94eb857d97e4b3f28c9c/build_O2_fullCI_slc9/pretty.html).

--- a/Common/Constants/include/CommonConstants/PhysicsConstants.h
+++ b/Common/Constants/include/CommonConstants/PhysicsConstants.h
@@ -76,6 +76,7 @@ enum Pdg {
   kTriton = 1000010030,
   kHelium3 = 1000020030,
   kAlpha = 1000020040,
+  kLithium4 = 1000030040,
   kHyperTriton = 1010010030,
   kHyperHydrogen4 = 1010010040,
   kHyperHelium4 = 1010020040,
@@ -129,10 +130,11 @@ constexpr double MassDeuteron = 1.87561294257;
 constexpr double MassTriton = 2.80892113298;
 constexpr double MassHelium3 = 2.80839160743;
 constexpr double MassAlpha = 3.7273794066;
-constexpr double MassHyperTriton = 2.99131;
-constexpr double MassHyperHydrogen4 = 3.9226;
-constexpr double MassHyperHelium4 = 3.9217;
-constexpr double MassHyperHelium5 = 4.841;
+constexpr double MassLithium4 = 3.7513;
+constexpr double MassHyperTriton = 2.991134;
+constexpr double MassHyperHydrogen4 = 3.922434;
+constexpr double MassHyperHelium4 = 3.921728;
+constexpr double MassHyperHelium5 = 4.839961;
 constexpr double MassHyperHelium4Sigma = 3.995;
 
 /// \brief Declarations of masses for particles in ROOT PDG_t

--- a/Common/Constants/include/CommonConstants/make_pdg_header.py
+++ b/Common/Constants/include/CommonConstants/make_pdg_header.py
@@ -131,6 +131,7 @@ class Pdg(Enum):
     kTriton = 1000010030
     kHelium3 = 1000020030
     kAlpha = 1000020040
+    kLithium4 = 1000030040
     kHyperTriton = 1010010030
     kHyperHydrogen4 = 1010010040
     kHyperHelium4 = 1010020040

